db/schema.rb

```diff
@@ -11,6 +11,43 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 0) do
+ActiveRecord::Schema.define(version: 20160112013657) do
+
+  create_table "projects", force: true do |t|
+    t.float    "gain",       limit: 24
+    t.float    "bpm",        limit: 24
+    t.string   "time_sig"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  create_table "sound_objects", force: true do |t|
+    t.integer  "track_id"
+    t.integer  "project_position", limit: 8
+    t.integer  "sound_id"
+    t.integer  "sound_position",   limit: 8
+    t.integer  "duration",         limit: 8
```

Comment:
Can we store the amount of ticks cropped from the end of a file, rather than the duration? We can reference total duration of an audio buffer from the Sound record